src/macaron/config/defaults.ini

```diff
@@ -295,6 +295,20 @@ deploy_arg =
 [builder.poetry.ci.deploy]
 github_actions = pypa/gh-action-pypi-publish
 
+# This is the spec for trusted Docker build tool usages.
+[builder.docker]
+entry_conf =
+# Also account for if there's multiple dockerfiles, e.g. test.Dockerfile
+build_configs =
+    Dockerfile
+    Dockerfile.*
```

Comment:
Hmm, the [`file_exists`](https://github.com/oracle/macaron/blob/staging/src/macaron/slsa_analyzer/build_tool/base_build_tool.py) method is not supposed to expand the wildcard pattern from file names (at least that is what I have been assuming for a long time). 
I guess it will expand the wild card pattern because at [these lines](https://github.com/oracle/macaron/blob/ae2c859ba13451bff35dc7986a029a8151006ca0/src/macaron/slsa_analyzer/build_tool/base_build_tool.py#L35C1-L36C57) we are concatenating the `file_name` variable (which is the direct input from `defaults.ini`) to the call of `glob.iglob`. 
@behnazh-w Is this behavior expected ? If yes, I guess we should improve the documentation in the `files_exist` method to notice about this behavior.

Reply:
Yes, this behavior is expected. Based on the [`glob.iglob`](https://docs.python.org/3/library/glob.html#glob.iglob), which follows [`glob.glob`](https://docs.python.org/3/library/glob.html#glob.glob):
>pathname can contain shell-style wildcards

We could add a note to [`file_exists`](https://github.com/oracle/macaron/blob/staging/src/macaron/slsa_analyzer/build_tool/base_build_tool.py) but that's expected of `glob.iglob` to handle wildcards anyway.